core/src/main/java/io/questdb/griffin/engine/groupby/InterpolationUtil.java

```diff
@@ -158,15 +158,15 @@ static void interpolateBoundary(
             MapValue mapValue1,
             MapValue mapValue2,
             boolean isEndOfBoundary
-    ) throws SqlException {
```

Comment:
Switching to cairo exception makes error message disappear from console/pg wire . 
Is that intentional ?

Reply:
No, it's not intentional. Let me fix that.

Reply:
Fixed in d2df1153e7d1cc7574260c36c9f21ae21686ed22